### PR TITLE
api gateway refactor

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
# Description

refactor api-gateway microservice

## Related Issue

fixes#114

## Motivation and Context
1. Implemented es6 import/export ( as in some files 'require' and 'module.exports' - es5 feature are implemented).
2. Replaced arrow function with normal function where it is not needed.
     > function have more than one line body.
     > expecting function to does not have it's own scope
3. File rename
     > used dash separation instead of underscore;
     > renamed files to appropriate entity ; such as ('nft_commitment.' changed to 'nft-commitment.' , and 'ft_commitment' to 'ft-commitment').
     > used smaller case of api-gateway microservice. So, now 'api-gateway' instead of 'API-Gateway'

## How Has This Been Tested

1. by running integration test
2. by using UI.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api-gateway/src/rest/db.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+      method: 'PATCH',
+      json: true,
+      headers: { name },
+      body,
+    };
+    return requestWrapper(options);
+  },
+
+  // insert fungible token commitment transaction info in ft_commitment_transaction collection
+  addCoinTransaction({ name }, body) {
```
Comment (ChaitanyaKonda) [inline api-gateway/src/rest/db.js]: Terminology coin needs to be updated everywhere. To FT.

**Comment by ChaitanyaKonda** [comment]
Can be reviewed now as #112 is merged

File: api-gateway/src/rest/db.js (hunk 1 above)
Comment (LijuJoseJJ) [reply api-gateway/src/rest/db.js]: we are planing to this in different PR with Restful changes

**Comment by ChaitanyaKonda** [comment]
Reviewed. Tests passed

**Pull request merged**